_source/api/logzio-public-api.yml

```diff
@@ -3895,13 +3896,18 @@ definitions:
 
   
   # ::::: ACCOUNTS DEFINITIONS
-  WhoAmI:
+  WhoAmIV2Response:
     type: object
     properties:
+      accountID:
+        type: integer
+        format: int32
+        description: Account ID 
+        example: 123456
       accountName:
         type: string
         description: Name of the account
-        example: Jean Valjean
```

Comment:
this would of course require the id in the example to be 24601 😄

Reply:
Since it's an account name - why not "Deployment Staging account"

Reply:
i dont mind what the name is in the end, but if we are going for Jean Valjean then the id has got to be 24601 😄

Reply:
https://www.youtube.com/watch?v=nXhofhspBd4

Reply:
I completely LURVE your suggestion!
Now that I get it.